Source/WebCore/loader/DocumentThreadableLoader.cpp

```diff
@@ -681,13 +682,15 @@ void DocumentThreadableLoader::loadRequest(ResourceRequest&& request, SecurityCh
             }
         }
     }
-    didReceiveResponse(identifier, response);
+
+    const auto* timing = response.deprecatedNetworkLoadMetricsOrNull();
+    auto resourceTiming = ResourceTiming::fromSynchronousLoad(requestURL, m_options.initiatorType, loadTiming, timing ? *timing : NetworkLoadMetrics::emptyMetrics(), response, securityOrigin());
+
+    didReceiveResponse(identifier, WTFMove(response));
```

Comment:
Why are we swapping the order of operations here?

Reply:
This is just so we can retrieve the resourceTiming value before we move the `response` elsewhere. If we don't, we get a bad load.